connectors/sources/azure_blob_storage.py

```diff
@@ -105,7 +106,7 @@ def get_default_configuration(cls):
                 "validations": [
                     {"type": "less_than", "constraint": MAX_CONCURRENT_DOWNLOADS + 1}
                 ],
-                "value": MAX_CONCURRENT_DOWNLOADS,
+                "value": None,
```

Comment:
if we always have to set `value` to `None`, what if we just remove the `value` key in that case for the default config -- seems to just be noise at that point or am I missing something ?

Reply:
That's what I'll be doing! This PR is only complete for the DB ones.
(Also this is a draft please don't review it yet haha)